src/naksu/ui.go

```diff
@@ -47,12 +56,35 @@ var backupLabel *ui.Label
 
 var backupMediaPath []string
 
+// Destroy Confirmation Window
+var destroyWindow *ui.Window
+
+var destroyButtonDestroy *ui.Button
+var destroyButtonCancel *ui.Button
+
+var destroyBox *ui.Box
+
+var destroyInfoLabel [5]*ui.Label
+
+// Remove Confirmation Window
+var removeWindow *ui.Window
+
+var removeButtonRemove *ui.Button
+var removeButtonCancel *ui.Button
+
+var removeBox *ui.Box
+
+var removeInfoLabel [5]*ui.Label
+
+
 func createMainWindowElements() {
 	// Define main window
-	buttonStartServer = ui.NewButton(xlate.Get("Start Stickless Exam Server"))
-	buttonGetServer = ui.NewButton("Install or update Abitti Stickless Exam Server")
-	buttonSwitchServer = ui.NewButton("Install or update Stickless Matriculation Exam Server")
-	buttonMakeBackup = ui.NewButton("Make Stickless Exam Server Backup")
+	buttonStartServer = ui.NewButton(xlate.Get("Start Exam Server"))
```

Comment:
This seems out of place? If I recall correctly the translations are updated later on so `xlate.Get(...)` is not necessary here